PublicDateAtUSN: 2024-07-09 14:00:00 UTC
Candidate: CVE-2024-38875
PublicDate: 2024-07-10 05:15:00 UTC
CRD: 2024-07-09 14:00:00 UTC
References:
 https://www.cve.org/CVERecord?id=CVE-2024-38875
 https://ubuntu.com/security/notices/USN-6888-1
 https://ubuntu.com/security/notices/USN-6888-2
Description:
 An issue was discovered in Django 4.2 before 4.2.14 and 5.0 before 5.0.7.
 urlize and urlizetrunc were subject to a potential denial of service attack
 via certain inputs with a very large number of brackets.
Ubuntu-Description:
Notes:
 alexmurray> upstream advises that only versions 4.2, 5.0 and 5.1 (plus main
  development branch) are affected but it is likely earlier versions may
  also be affected but upstream do not mention this as they are no longer
  maintained by them
Mitigation:
Bugs:
Priority: medium
Discovered-by: Elias Myllym√§ki
Assigned-to:
CVSS:
 cisa-adp: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]

Patches_python-django:
upstream_python-django: needs-triage
trusty/esm_python-django: ignored (end of ESM support, was needs-triage)
esm-infra-legacy/trusty_python-django: needs-triage
esm-infra/xenial_python-django: needs-triage
esm-infra/bionic_python-django: released (1:1.11.11-1ubuntu1.21+esm5)
focal_python-django: released (2:2.2.12-1ubuntu0.23)
esm-infra/focal_python-django: released (2:2.2.12-1ubuntu0.23)
jammy_python-django: released (2:3.2.12-2ubuntu1.12)
mantic_python-django: released (3:4.2.4-1ubuntu2.3)
noble_python-django: released (3:4.2.11-1ubuntu1.1)
oracular_python-django: not-affected (3:4.2.14-1)
plucky_python-django: not-affected (3:4.2.14-1)
devel_python-django: not-affected (3:4.2.14-1)
